README.md

```diff
@@ -4,7 +4,38 @@ React hooks for fetching data from gtfs sources.
 
 ## Usage
 
-TODO: Describe.
+There are two hooks which this package provides. `useGtfsSchedule` and `useGtfsRealtime`. Both take two arguments, a resolver and a timeout.
+
+The resolver must be an async function which takes no arguments and returns an array buffer containing the raw data for the hook to process.
+For the schedule hook, this should be the raw zip data coming from your GTFS Schedule endpoint.
+For the realtime hook, this should be the protobuf binary data from your GFTS Realtime Alerts endpoint.
+
+The timeout defines how often the resolver will be called to fetch new data. This should be defined in milliseconds.
+
+The following is suitable for most purposes:
+```js
+const fetchGtfsSchedule = useCallback(async () => {
+  const response = await fetch(gtfsScheduleUrl)
+  return new Uint8Array(await response.arrayBuffer())
+}, [gtfsScheduleUrl])
+
+const gtfsSchedule = useGtfsSchedule(fetchGtfsSchedule, 1000 * 60 * 60 * 24)
+
+const fetchGtfsRealtimeAlerts = useCallback(async () => {
+  const response = await fetch(gtfsRealtimeAlertsUrl)
+  return new Uint8Array(await response.arrayBuffer())
+}, [gtfsRealtimeAlertsUrl])
+
+const gtfsRealtimeAlerts = useGtfsRealtime(fetchGtfsRealtimeAlerts, 1000 * 30)
+```
+
+The object returned by the GTFS Schedule hook follows the [GTFS Schedule Standard][gtfs-schedule-standard]'s structure, except for all field/attribute names being camel cased. Note that our implementation
+operates without checking if the standard is being met. Thus any other files in the zip will be parsed as a CSV and added to the returned object.
```

Comment:
We should probably also mention that the individual files will load asynchronously as well - i.e. you might get back `{}` while the individual files are being read.